automap/src/comm_layer/pcp_pmp_common/windows_specific.rs

```diff
@@ -0,0 +1,401 @@
+// Copyright (c) 2019-2021, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+#![cfg(target_os = "windows")]
+
+use crate::comm_layer::pcp_pmp_common::FindRoutersCommand;
+use crate::comm_layer::AutomapError;
+use std::net::IpAddr;
+use std::str::FromStr;
+
+pub fn windows_find_routers(command: &dyn FindRoutersCommand) -> Result<Vec<IpAddr>, AutomapError> {
+    match command.execute() {
+        Ok(stdout) => {
+            // Arrange to split output into line pairs that we can zip together
+            let mut firsts = stdout.split(&['\n', '\r'][..]).collect::<Vec<&str>>();
+            firsts.push("");
+            let mut seconds = vec![""];
+            seconds.extend(firsts.clone());
+            let addresses = firsts
```

Comment:
I don't have energy to break it down but what caught me was seeing:

`firsts` and then `seconds` ... when the zip is set

and 

`second, first` when call the values inside the iterators 

maybe it's really vica versa